src/commands/latency-doctor.json

```diff
@@ -1,7 +1,7 @@
 {
     "DOCTOR": {
         "summary": "Return a human readable latency analysis report.",
-        "complexity": "PATCH__TBD__42__",
+        "complexity": "NA",
```

Comment:
is "NA" now part of the "protocol"? maybe we better leave empty?
@itamarhaber WDYT?

p.s. isn't it an O(1) command?

Reply:
"complexity" is just free text, doesn't have to comply with any rules

we left "NA" so that it'll be easy to grep all the places that are missing this information

Reply:
well, ok. but what's the difference between `NA`, a missing field, or one set to `""`?

Reply:
it should at least be uniform, i feel that a missing field is better if we don't have anything to say